meeting_minutes/220714_SARIF_TC_62.md

```diff
@@ -0,0 +1,157 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                 |
+|:-----------|:----------|:------------------|:------------------------|
+| Aditya     | Sharad    | Microsoft         | Member                  |
+| Aharon     | Abadi     | WhiteSource       | Voting Member           |
```

Comment:
Aharon lost his rights last time according to https://github.com/oasis-tcs/sarif-spec/blob/master/meeting_minutes/220630_SARIF_TC_61.md#173-members-losing-voting-rights-if-they-have-not-joined-this-meeting-by-the-time-it-ends
```suggestion
| Aharon     | Abadi     | WhiteSource       | Member |
```